src/Symfony/Component/Form/Extension/Core/Type/DateIntervalType.php

```diff
@@ -192,6 +195,21 @@ public function configureOptions(OptionsResolver $resolver)
             return array_fill_keys($timeParts, $placeholder);
         };
 
+        $labelsNormalizer = function (Options $options, array $labels) {
+            return array_replace(array(
+                'years' => null,
+                'months' => null,
+                'days' => null,
+                'weeks' => null,
+                'hours' => null,
+                'minutes' => null,
+                'seconds' => null,
+                'invert' => 'Negative interval',
+            ), array_filter($labels, function ($label) {
```

Comment:
@HeahDude : Do you think gathering the labels under a single option is a good idea or should I opt back for `label_x` options?

#21002 is suggesting to translate `DateTimeType` subtypes labels and uses dedicated options. I guess it's coherent regarding other options actually, even if it'll looks ugly in the case of the `DateIntervalType`. But I guess we should be consistent. WDYT?